src/plugins/conditionals/conditionals.c

```diff
@@ -209,7 +209,7 @@ static CondResult evalCondition (const Key * curKey, const char * leftSide, Comp
 			}
 			if (elektraRealloc ((void **) &compareTo, endPos - rightSide) < 0)
 			{
-				ELEKTRA_SET_ERROR (87, parentKey, "Out of memory");
+				ELEKTRA_SET_OUT_OF_MEMORY_ERROR (parentKey, "Out of memory");
```

Comment:
Maybe we make ELEKTRA_SET_OUT_OF_MEMORY_ERROR to not accept arguments: or we pass what was resized with which size.

Reply:
I have thought about that too since since there is already a macro in [kdbmacros.h](https://github.com/ElektraInitiative/libelektra/blob/d3d98a8216430e40bdcc09004c668f49e38f3e92/src/include/kdbmacros.h#L64). I don't think though that does yield a considerable benefit (consistency, additional information) but might confuse developers that all macros behave identically but OUT_OF_MEMORY suddenly does not.

Reply:
Yes, if OUT_OF_MEMORY would be the only one it might be confusing. But I think it is okay in general for macros of different categories to behave differently (or to have specializations that behave differently, see #2697). To pass an argument, that does not provide any information, does not make sense. As alternative we could pass as argument how much memory we tried to allocate: but this is a lot of work with little benefit.